packages/devtools_app/lib/src/screens/debugger/controls.dart

```diff
@@ -54,7 +56,7 @@ class _DebuggingControlsState extends State<DebuggingControls>
           const SizedBox(width: denseSpacing),
           BreakOnExceptionsControl(controller: controller),
           const SizedBox(width: denseSpacing),
```

Comment:
group this SizedBox with the CodeCoverageToggle 
```
if (isVmAp) ...[
 ...
],
```

Reply:
done!